app/src/main/java/com/mobiledevelopmentfoodapp/MainActivity.kt

```diff
@@ -79,7 +78,7 @@ fun ButtonBar() {
 @Composable
 fun Menu() {
     val items = listOf("A", "B", "C", "D", "E", "F")
-    var countryName : String by remember { mutableStateOf("Menu Categories") }
+    var restaurantName : String by remember { mutableStateOf("Menu Categories") }
```

Comment:
Renamed this variable from countryName to restrauntName as it is a much more relevant name for this project.

Reply:
Yes, good.